src/context/AuthContext.tsx

```diff
@@ -65,24 +67,55 @@ export const AuthProvider: FC<{ children: ReactNode }> = ({ children }) => {
             .catch((error) => {
                 throw new Error('Error during logout:', error);
             })
+        // return await makeRequest('POST', `logout`, {})
+        //     .then(() => {
+        //         localStorage.removeItem('userId');
+        //         localStorage.removeItem('accessToken');
+        //         setUser(undefined);
+        //     })
+        //     .catch((error) => { throw new Error(error) });
     }
 
     const isAuth = async (userId: string) => {
```

Comment:
nazwa powinna byc bardziej login lub loginUser. isAuth to brzmi jakby to mial bys jakis warunek, na pewno nie przypomina nazwa funkcji odpowiedzialnej za logowanie